File: Python/tests/test_pstRT_rlddm6.py
import pytest

from hbayesdm.models import pstRT_rlddm6


def test_pstRT_rlddm6():
    _ = pstRT_rlddm6(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_pstRT_rlddm6.R
context("Test pstRT_rlddm6")
library(hBayesDM)

test_that("Test pstRT_rlddm6", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(pstRT_rlddm6(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
